astropy/utils/iers/iers.py

```diff
@@ -983,8 +983,11 @@ def auto_open(cls, files=None):
         that expires more than 180 - `~astropy.utils.iers.Conf.auto_max_age`
         after the present.
         """
-        good_enough = cls._today() + TimeDelta(180-_none_to_float(conf.auto_max_age),
-                                               format='jd')
+        if conf.auto_max_age is not None:
```

Comment:
I think one would still want to get the best file rather than the first that is available so I would write
```
offset = 180 - (30 if conf.auto_max_age is None else conf.auto_max_age)
good_enough = cls._today() + TimeDelta(180-offset, format='jd')
```

Reply:
Done.